vs_utils/utils/grid_featurizer.py

```diff
@@ -30,41 +40,86 @@ def generate_random_unit_vector():
   d. output u
   '''
 
-  theta = np.random.uniform(low = 0.0, high = 2*np.pi)
-  z = np.random.uniform(low = -1.0, high = 1.0)
-  u = np.array([np.sqrt(1-z**2)*np.cos(theta), np.sqrt(1-z**2)*np.sin(theta), z])
+  theta = np.random.uniform(low=0.0, high=2 * np.pi)
+  z = np.random.uniform(low=-1.0, high=1.0)
+  u = np.array([np.sqrt(1 - z**2) * np.cos(theta),
+          np.sqrt(1 - z**2) * np.sin(theta), z])
   return(u)
 
+
 def generate_random_rotation_matrix():
   '''
    1. generate a random unit vector, i.e., randomly sampled from the unit 3-sphere
-      a. see function generate_random_unit_vector() for details
+    a. see function generate_random_unit_vector() for details
     2. Generate a second random unit vector thru the algorithm in (1), output v
-      a. If absolute value of u \dot v > 0.99, repeat. This is important for numerical stability 
-        (intuition: we want them to be as linearly independent as possible or else the 
-          orthogonalized version of v will be much shorter in magnitude compared to u. I assume
-          in Stack they took this from Gram-Schmidt orthogonalization?)
-      b. v' = v - (u \dot v)*u, i.e. subtract out the component of v that's in u's direction 
-      c. normalize v' (this isn't in Stack but I assume it must be done)
-    3. find w = u \cross v' 
+    a. If absolute value of u \dot v > 0.99, repeat. This is important for numerical stability
```

Comment:
I think autopep8 might have messed this part up. The original extra indentation for a, b, c looked nicer.